src/app/lib/supabase.js

```diff
@@ -0,0 +1,10 @@
+import { createClient } from '@supabase/supabase-js';
+
+const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL;
+const supabaseKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY;
+
+if (!supabaseUrl || !supabaseKey) {
+  throw new Error('Supabase URL and API key are required.');
+}
+
+export const supabase = createClient(supabaseUrl, supabaseKey);
```

Comment:
everything here is great. Using env vars, validating the presence of said env vars, exporting as a named export (I have a thing for named exports). 👍